dashboard/lib/school_info_deduplicator.rb

```diff
@@ -70,7 +58,15 @@ def process_school_info_attributes(school_info_attr)
       :school_name,
       :full_address,
       :validation_type
-    )
+    ]
+
+    attr.slice!(*relevant_attributes)
```

Comment:
I wonder if we can save some folks a quick google and add a brief comment that this only leaves the provided keys intact?

Reply:
Added a comment.